app/src/components/common/footer/index.tsx

```diff
@@ -10,13 +10,22 @@ const FooterStyled = styled.footer`
   padding-top: 50px;
 `
 
-const AFooter = styled.a`
+const FooterCss = css`
   align-items: center;
   display: flex;
   justify-content: center;
   text-decoration: none;
 `
 
+const AFooterGeco = styled.a`
```

Comment:
Can we rename this to `FooterGeco`? Why the `A`?

Reply:
I think is for the 'anchor' tag.  Anyway I update it to a more consistent name.